src/vm-repair/azext_vm_repair/_validators.py

```diff
@@ -41,7 +41,7 @@ def validate_create(cmd, namespace):
     if namespace.repair_vm_name:
         _validate_vm_name(namespace.repair_vm_name, is_linux)
     else:
-        namespace.repair_vm_name = ('repair-' + namespace.vm_name)[:15]
+        namespace.repair_vm_name = ('repair-' + namespace.vm_name)[:14] + '_'
```

Comment:
May I know why "-" is allowed after "repair" but not at the end?

Reply:
Its how Azure resource naming is set up. It cannot end with a hyphen.